.gitattributes

```diff
@@ -0,0 +1 @@
+src/main/resources/org/triplea/client/ui/javafx/images/* filter=lfs diff=lfs merge=lfs -text

```

Comment:
The benefit of having a parallel folder (e.g. _src/main/resources-large_) tracked via LFS is that it doesn't force us to store _all_ images for this package using LFS.  I'm assuming there could be KiB-sized images in this folder that wouldn't make sense to track via LFS.  Then you can place the small images in _src/main/resources/org/triplea/client/ui/javafx/images_ and the large images in _src/main/resources-large/org/triplea/client/ui/javafx/images_.  We just need to define what the threshold is between "small" and "large."

Reply:
Why not track all images with git LFS? I'm thinking of the rule of thumb that no binaries should be stored in git, seems just as well to me to track even small images with all the other ones.

Reply:
>  I'm thinking of the rule of thumb that no binaries should be stored in git

Remember, it's _Large_ File Storage, not _Binary_ File Storage. :smile:  It may not be optimized for all use cases.

In git-lfs/git-lfs#282, you can find several comments along the lines of [this one](https://github.com/git-lfs/git-lfs/issues/282#issuecomment-307989798):

> Tracking small files in LFS slows down LFS pull considerably, and when working with hundreds of GB of data, that's a really big deal.

I expect this complaint is related to having to fetch hundreds or thousands of files from LFS, which may not be optimized for using a single connection to transfer multiple files.  We're probably not going to hit this threshold, but it's something worth considering.

Here's [another view](https://stackoverflow.com/questions/35787080/how-well-does-git-lfs-handle-small-files) on this issue that adds the dimension of how often the file changes.  However, the last comment indicates that LFS file retrieval is synchronous, which would confirm the observation in the above comment that many files in LFS can slow down pulls.

Reply:
That is good context, thank you. 
@ssoloff  Would it make any sense to zip small files together? Or really best to just check them in? We only strictly need them at packaging time, so we could use something like the `assets` repo, but that makes development a bit more cumbersome. If none of those are good ideas then I could be pretty easily convinced to "just check them in."  For reference, how small is a small file roughly? say less than <100kB?

Reply:
> Would it make any sense to zip small files together? Or really best to just check them in?

I'm leaning towards just checking them in.

> how small is a small file roughly? say less than <100kB?

I think 100 KiB is probably a good threshold to start with.

Reply:
> I'm leaning towards just checking them in.

Now leaning further...  I just did a pull after this PR was merged, and saw what the commenter in the other thread was talking about.  A nice 1-2 second pause as it opened a connection to download each image.